lxd/api.go

```diff
@@ -98,6 +99,12 @@ func restServer(d *Daemon) *http.Server {
 		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
 			http.Redirect(w, r, "/ui/", http.StatusMovedPermanently)
 		})
+	} else {
+		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "text/html")
+			w.WriteHeader(http.StatusNotImplemented)
+			fmt.Fprint(w, `The UI is not enabled. For instructions to enable it check: <a href="https://documentation.ubuntu.com/lxd/en/latest/howto/access_ui/">How to access the LXD web UI</a>`)
```

Comment:
Lets use a constant for this so it can be included in both places rather than being duplicated.
Thanks